website/addons/googledrive/routes.py

```diff
@@ -6,59 +6,18 @@
 
 from . import views
 
-# Routes that use the web renderer
-auth_routes = {
-    'rules': [
-
-        ##### OAuth #####
-
-        Rule(
-            ['/oauth/connect/googledrive/'],
-            'get',
-            views.auth.googledrive_oauth_start,
-            json_renderer,
-            endpoint_suffix='_user'
-        ),
-
-        Rule(
-            ['/oauth/callback/googledrive/'],
-            'get',
-            views.auth.googledrive_oauth_finish,
```

Comment:
It looks like this route is still being used by `GoogleAuthClient.start()` and  `GoogleAuthClient.finish()`: 
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/googledrive/client.py#L19
https://github.com/CenterForOpenScience/osf.io/blob/develop/website/addons/googledrive/client.py#L31